sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
No, the main model is sale.order, not sale.order.line, that's why I ask for sale_order.py